armi/utils/densityTools.py

```diff
@@ -52,6 +56,10 @@ def getMassFractions(numberDensities):
     """
     Convert number densities (#/bn-cm) into mass fractions.
 
+    .. impl:: Get number densities
+        :id: I_ARMI_UTIL_MASS2N_DENS1
+        :implements: R_ARMI_UTIL_MASS2N_DENS
+
```

Comment:
I think we can delete this one.  I believe the `getNDensFromMasses()` function above is the only function this Req is pointing at, and you already have that covered.